src/resources/Entity/index.ts

```diff
@@ -55,6 +56,27 @@ export interface IEntityWithdrawConsentOpts {
   type: 'withdraw',
   reason: 'entity_withdrew_consent' | null,
 };
+export interface IEntity {
```

Comment:
Is this a new pattern you're going for where main interface is in `index` instead of `type`? Saw this for account too, fine either way just wondering if there was a reason for the change

Reply:
yep! i noticed a circular dependency issue in the python library and realized this had the same issue, so I move it